action.yml

```diff
@@ -9,6 +9,14 @@ inputs:
     default: debian/artifacts
     description: Directory path relative to $GITHUB_WORKSPACE to place the built packages in
     required: true
+  use-debuild:
+    default: 'false'
+    required: true
+    description: If true, debuild is used to create the debian package. Otherwise, dpkg-buildpackage is used
+  debuild-opts:
+    default: '-b -rfakeroot -us -uc'
```

Comment:
I don’t want to assume too many—well, any— default options to dpkg-buildpackage (or debuild) in this action. That’s one of the reasons why I created it in the first place. As long as it makes sense, it should behave as similar as possible to as when you call the build tool(s) manually in your terminal.

Also:

`-rfakeroot` is the default of both debuild and dpkg-buildpackage, this can be left out.

If `-b` is given, it apparently can never be overwritten with anything else, e.g. `-F` (= `--build=full`). debuild/dpkg-buildpackage just fails when both are passed together. So having `-b` in here would restrict the action to one use-case.